### ISSUE TITLE
Impostor (private server) support

### ISSUE THREAD

**Issue by Permanently** [polusgg/Town-Of-Us#72]
[Impostor](https://github.com/Impostor/Impostor) is a program, aiming to be a platform for Among Us private servers. One thing I quickly found out is that this mod and Impostor are both incompatible with each other.

The Impostor authors [have said](https://github.com/Impostor/Impostor/issues/319#issuecomment-778763793) that they won't go out of their way to support mods, and to consult mod authors instead, so thought this'd be worth trying to ask for.

**Comment by VanillaChan6571** [comment]
Disable Anti-Cheat in the config to have mods.

**Comment by Permanently** [comment]
When I opened this issue, I was extremely tired and failed to describe specifically what the problem is.

The issue isn't with Impostor, and there's nothing I can do with the server that will change the outcome, since this is a client-side problem. Whenever Town Of Us is installed and Impostor patches the game to route to a private server, it instead selects the official Asia servers, and this is only with Town Of Us installed. Whenever I've tried any other mods, there's been absolutely no issues.
